### PR TITLE
Fix swagger ui payload not getting sent error in POST request

### PR DESCRIPTION
**Pull request by CrowleyRajapakse**
## Issue
fixes https://github.com/wso2/product-apim/issues/7336

## Methodology

Using swagger-ui-react library which resolves the payload getting lost for OASv3 definitions ,instead of swagger-ui library and remove the custom implementations needed for the previous library.

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/public/css/main.css [hunk 1]
```diff
@@ -1,420 +1,587 @@
...
+
 @font-face {
-  font-family: 'Open Sans';
-  font-style: italic;
-  font-weight: 400;
-  src: local('Open Sans Italic'), local('OpenSans-Italic'), url(../fonts/Open_Sans/mem6YaGs126MiZpBA-UFUK0ddc1GAK6bt6o.woff2) format('woff2');
-  unicode-range: U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;
+    font-family: 'Open Sans';
+    font-style: italic;
+    font-weight: 400;
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/public/css/main.css]: @CrowleyRajapakse , shall we remove this unrelated formation changes and put a comment in our hack 😃

Comment (CrowleyRajapakse) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/public/css/main.css]: resolved
